src/Elliptic/Systems/Elasticity/Equations.hpp

```diff
@@ -106,22 +106,16 @@ struct Fluxes {
                  domain::Tags::Coordinates<Dim, Frame::Inertial>>;
   using volume_tags = tmpl::list<Tags::ConstitutiveRelationBase>;
   static void apply(
-      const gsl::not_null<tnsr::IJ<DataVector, Dim>*> flux_for_displacement,
+      gsl::not_null<tnsr::IJ<DataVector, Dim>*> flux_for_displacement,
       const ConstitutiveRelations::ConstitutiveRelation<Dim>&
           constitutive_relation,
       const tnsr::I<DataVector, Dim>& coordinates,
-      const tnsr::ii<DataVector, Dim>& strain) noexcept {
-    primal_fluxes(flux_for_displacement, strain, constitutive_relation,
-                  coordinates);
-  }
-  static void apply(
-      const gsl::not_null<tnsr::Ijj<DataVector, Dim>*> flux_for_strain,
-      const ConstitutiveRelations::ConstitutiveRelation<
-          Dim>& /*constitutive_relation*/,
-      const tnsr::I<DataVector, Dim>& /*coordinates*/,
-      const tnsr::I<DataVector, Dim>& displacement) noexcept {
-    auxiliary_fluxes(flux_for_strain, displacement);
-  }
+      const tnsr::ii<DataVector, Dim>& strain) noexcept;
+  static void apply(gsl::not_null<tnsr::Ijj<DataVector, Dim>*> flux_for_strain,
+                    const ConstitutiveRelations::ConstitutiveRelation<Dim>&
+                        constitutive_relation,
+                    const tnsr::I<DataVector, Dim>& coordinates,
+                    const tnsr::I<DataVector, Dim>& displacement) noexcept;
   // clang-tidy: no runtime references
   void pup(PUP::er& /*p*/) noexcept {}  // NOLINT
```

Comment:
[optional] while you squash you could move the `pup`. Again, up to you :)

Reply:
The pup function will go away soon, so I'll just leave it as is and delete ASAP